src/cycleCheck.es6.js

```diff
@@ -0,0 +1,11 @@
+function cycleCheck (list, keys, all = false) {
+  var err = false;
+  keys[ all ? "map":"some"]((key) => {
+    if (typeof list[key] === undefined) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://www.codacy.com/app/lagora-franck/proc-edu-ria/file/2572032743/issues/source?bid=3286583&fileBranchId=3298273#l4)